frontend-vue3/src/components/form/FormDatePicker.vue

```diff
@@ -32,17 +33,18 @@ import FormLabel from "@/components/form/FormLabel.vue";
 import FormHelp from "@/components/form/FormHelp.vue";
 import FormError from "@/components/form/FormError.vue";
 
-export type SelectOptions = { value: any; label: string }[];
-
 export interface DatePickerProps {
   name: string;
   label?: string;
   help?: string;
   placeholder?: string;
   required?: boolean;
+  string?: boolean; // whether to use string or date for the model value
```

Comment:
for booleans I lowkey prefer prefixing with is/can to make it a predicate verb so `isRequired` `isString` etc. thoughts?

Reply:
Yeah that's an improvement. May be a better way to express these in particular since the names don't indicate all that well what they do

Reply:
changed `required` -> `indicateRequired` since it just draws an asterisk, removed `string` in favor of only dealing with `Date` in the datepicker